productListing.html

```diff
@@ -145,7 +145,104 @@ <h4 class="products_sidebar_title">Rating</h4>
                 </div>
             </div>
 
-            <div class="productlisting_products"></div>
+            <div class="card_container">
+                <div class="card-basic card_custom_width">
+                  <div class="badge-container">
+                    <img
+                      class="card-img"
+                      src="https://d3cif2hu95s88v.cloudfront.net/blog/wp-content/uploads/2021/05/04150624/Spider-plant.jpg"
+                    />
+                    <span class="card-withBadge">New</span>
+        
+                    <div class="">
+                      <h3 class="card-heading pri_clr">Aleovera Indoor Plant</h3>
+                      <small class="card-subHeading">Outdoor</small>
+                    </div>
+        
+                    <p class="card-description text-bold pri_clr">₹ 10,999</p>
+                  </div>
+        
+                  <div class="card-footer">
+                    <button class="btn custom_btn">
+                      <span class="icon"><i class="fa fa-shopping-cart"></i></span> Add
+                      to Cart
+                    </button>
+                  </div>
+                </div>
+        
+                <div class="card-basic card_custom_width">
+                  <div class="badge-container">
+                    <img
+                      class="card-img"
+                      src="https://pyxis.nymag.com/v1/imgs/4a7/29b/4ef2f518834dd81d8ceb374caa54fd5171-22-plant-gifts.rsquare.w700.jpg"
+                    />
```

Comment:
Add alt tag to img

Reply:
Did the changes, thanks for the feedback